c2corg_ui/static/js/user.js

```diff
@@ -100,4 +100,12 @@ app.UserController.prototype.logout = function() {
 };
 
 
+/**
+ * @export
+ */
+app.UserController.prototype.hasEditRightsOuting = function(users) {
+  this.auth.hasEditRightsOuting(users);
```

Comment:
That looks weird to me to have that kind of alias! Can't you directly use the function from the auth service?